sonar-project.properties

```diff
@@ -0,0 +1,28 @@
+sonar.projectKey=process-analytics_bpmn-visualization-js
+sonar.organization=process-analytics
+
+# This is the name and version displayed in the SonarCloud UI.
+sonar.projectName=bpmn-visualization
+sonar.projectVersion=0.8.0
```

Comment:
:warning: we have to make the version consistent with what we have in the package.json
We should automate this on version changes: we can do it in another PR, but as soon as possible, otherwise this will become a mess and I don't want we to have to manage this manually.

Reply:
I asked myself the same.
We need to change the version on the package.json, because it's corresponding to the last tag. But, here, we are on delopment on the next.
Maybe on the tag workflow, we need to change the current version, after the tag.

Reply:
My concern was about the version sync, but yes we can also address the version that stays with the tagged version.
Some projects (I didn't remember which one, but it was a big like jest) use `x.y.z-post` version after release, we could do the same

Reply:
We can manage it with #909